src/averageAges.js

```diff
@@ -14,13 +16,22 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
```

Comment:
let's not use `let` it can be unpredictable
```suggestion
  const men = people.filter(person => person.sex === 'm');
```